psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -276,7 +276,7 @@ def compute_hessian(self, molecule):
         molclone.fix_com(True)
 
         # Record undisplaced symmetry for projection of diplaced point groups
-        core.set_parent_symmetry(molecule.schoenflies_symbol())
+        core.set_global_option("PARENT_SYMMETRY", molecule.schoenflies_symbol())
```

Comment:
I think a `core.set_global_option("PARENT_SYMMETRY", "")` is needed after the displacements, to reset the variable and prevent unwanted mappings in subsequent calcs.